src/api/firebase.js

```diff
@@ -345,5 +345,7 @@ export function comparePurchaseUrgency(data) {
 		//sort alphabetically by name for items with the same urgency
 		if (itemA.name > itemB.name) return 1;
 		if (itemA.name < itemB.name) return -1;
+
+		return 1;
```

Comment:
Returning any number other than `0` may impact the sort order of the array, depending on the items being checked. Are we sure that returning `1` is what we want?

Reply:
Ah good catch! 0 is probably what we want